src/controllers/event.js

```diff
@@ -50,7 +51,7 @@ eventController.create = (req, res, next) => {
 
   const newEvent = new Event(newEventDetails)
   newEvent.save()
-  // TODO: .then(event => req.user.role !== roles.BASIC ? event : sendEmail(event))
+    .then(event => req.user.role !== roles.BASIC ? event : sendEmail(event))
```

Comment:
i don't see any problem here other than readability maybe?, if i had this in `if/else` and in ternary it's just going to do the same thing if the user is `BASIC` then sends an email then the `sendEmail` function would resolve with the event, if he's not then resolve with the event without sending the email. so whats the reason here to put this in an if statement instead of ternary ?! @des-des